### PR TITLE
2.0 preset for editor-dark-mode

### PR DESCRIPTION
**Pull request by TheColaber**
WL wanted me to pr this

### CODE REVIEW

**Comment by TheColaber** [comment]
image

**Comment by WorldLanguages** [comment]
@mxmou What do you think about having this as a preset? I'm asking because this seems like an obvious addition, yet I don't remember discussing this with you. You once mentioned having a "2.0 editor" addon, but not a preset like this.

**Comment by TheColaber** [comment]
The "2.0 editor" add-on would most likely change the positioning of things like the title or the categories but not change things like the color. Then we'd have a super preset that would have all add-ons that make the editor look like 2.0 including editor dark mode, block shape, block colors, and the 2.0 editor items and obviously much more.

**Comment by FunctionalMetatable** [comment]
It'd be nice to have the scratch 2.0 logo :P

**Comment by CST1229** [comment]
And #2010 too, which has been dead for months.

File: addons/editor-dark-mode/addon.json [hunk 1]
```diff
@@ -1002,6 +1002,34 @@
...
+        "accent": "#f5f5f5ff",
+        "input": "#ffffffff",
+        "workspace": "#dddedeff",
+        "categoryMenu": "#e6e8e8ff",
+        "palette": "#e6e8e8ff",
+        "fullscreen": "#ffffffff",
+        "stageHeader": "#f4f4f4ff",
+        "border": "#bdbdbeff",
+        "textShadow": false,
+        "dots": true,
```
Comment (WorldLanguages) [inline addons/editor-dark-mode/addon.json:1028]: Why `"dots": true`?

File: addons/editor-dark-mode/addon.json [hunk 2]
```diff
@@ -1002,6 +1002,34 @@
         "darkScrollbars": true
       }
     },
+    {
+      "name": "Scratch 2.0 Style",
```
Comment (WorldLanguages) [inline addons/editor-dark-mode/addon.json:1006]: Maybe a better name is `Scratch 2.0 colors`?

**Comment by WorldLanguages** [comment]
Waiting for mxmou for merge

File: addons/editor-dark-mode/addon.json (hunk 2 above)
Comment (mxmou) [reply addons/editor-dark-mode/addon.json:1006]: Why not just "Scratch 2.0"?

File: addons/editor-dark-mode/addon.json (hunk 1 above)
Comment (mxmou) [reply addons/editor-dark-mode/addon.json:1028]: Scratch 2.0 did have dots, they're just smaller and harder to see.

File: addons/editor-dark-mode/addon.json [hunk 3]
```diff
@@ -1002,6 +1002,34 @@
...
+      "id": "scratch2",
+      "description": "A theme based on Scratch 2.0's colors.",
+      "values": {
+        "page": "#ffffffff",
+        "primary": "#179fd7ff",
+        "highlightText": "#4d97ffff",
```
Comment (mxmou) [inline addons/editor-dark-mode/addon.json:1012]: Scratch 2.0 uses orange (`#fba939`) for hovered menu bar items and block categories. I think that should be the highlight text color.

File: addons/editor-dark-mode/addon.json [hunk 4]
```diff
@@ -1002,6 +1002,34 @@
...
+        "menuBar": "#9c9ea2ff",
+        "activeTab": "#fafafaff",
```
Comment (mxmou) [inline addons/editor-dark-mode/addon.json:1014]: I think white or `#e6e8e8` (top or bottom of 2.0's gradient) would be more appropriate.

File: addons/editor-dark-mode/addon.json [hunk 5]
```diff
@@ -1002,6 +1002,34 @@
...
+        "tab": "#f1f2f2ff",
+        "selector": "#ffffffff",
+        "selector2": "#e7e8e8ff",
```
Comment (mxmou) [inline addons/editor-dark-mode/addon.json:1017]: Scratch 2 uses `#e6e8e8` for both.

File: addons/editor-dark-mode/addon.json [hunk 6]
```diff
@@ -1002,6 +1002,34 @@
...
+        "selectorSelection": "#d0d0d0ff",
+        "accent": "#f5f5f5ff",
```
Comment (mxmou) [inline addons/editor-dark-mode/addon.json:1019]: I would use `#f2f2f2` (background of 2.0's paint and sound editors).

File: addons/editor-dark-mode/addon.json [hunk 7]
```diff
@@ -1002,6 +1002,34 @@
...
+        "input": "#ffffffff",
+        "workspace": "#dddedeff",
+        "categoryMenu": "#e6e8e8ff",
+        "palette": "#e6e8e8ff",
+        "fullscreen": "#ffffffff",
+        "stageHeader": "#f4f4f4ff",
```
Comment (mxmou) [inline addons/editor-dark-mode/addon.json:1025]: Same suggestion as `activeTab`.

File: addons/editor-dark-mode/addon.json [hunk 8]
```diff
@@ -1002,6 +1002,34 @@
...
+        "border": "#bdbdbeff",
```
Comment (mxmou) [inline addons/editor-dark-mode/addon.json:1026]: Scratch 2 uses two different border colors, but `#d0d1d2` is more common.

File: addons/editor-dark-mode/addon.json (hunk 3 above)
Comment (TheColaber) [reply addons/editor-dark-mode/addon.json:1012]: Hmm... it looks weird on all other places... 
image

Comment (mxmou) [reply addons/editor-dark-mode/addon.json]: What about `#1e9ed6` (used for hovered and selected icons)?

**Review by WorldLanguages — APPROVED** [review]
Waiting for mxmou approval

**Review by mxmou — APPROVED** [review]

**Comment by Endermaster3690** [comment]
> Waiting for mxmou approval

he approved just so you know

**Comment by mxmou** [comment]
> It'd be nice to have the scratch 2.0 logo :P

That could be part of the actual 2.0 appearance addon.

**Pull request merged**